packages/ckeditor5-dev-release-tools/lib/tasks/updateversions.js

```diff
@@ -59,6 +65,36 @@ module.exports = async function updateVersions( { packagesDirectory, version, cw
 	}
 };
 
+/**
+ * @param {String} cwd
+ * @param {Array.<String>} globPatterns
+ * @param {UpdateVersionsPackagesDirectoryFilter|null} packagesDirectoryFilter
+ * @returns {Promise.<Array.<String>>}
+ */
+async function getPackageJsonPaths( cwd, globPatterns, packagesDirectoryFilter ) {
+	const pkgJsonPaths = await glob( globPatterns, {
+		cwd,
+		absolute: true,
+		nodir: true
+	} );
+
+	if ( !packagesDirectoryFilter ) {
```

Comment:
*minor*

this if could be removed if the default value for `packagesDirectoryFilter` is `(val) => val`.

Reply:
The question is whether we accept no value passed as a callback. The `packagesDirectoryFilter ` variable is always passed, so it overrides the default value, always.

Reply:
```
globalThis.foo = ( text, displayFn = console.log ) => {
    displayFn( text )
}

foo('Test') // OK
foo('Test', null) // displayFn is not a function
```

Reply:
Let't not make any changes.